templates/home.html

```diff
@@ -150,17 +91,17 @@ <h5><a href="{% url 'resources' %}">Learn More</a></h5>
                 <hr>
             </div>
             <div id="column-wrapper">
-                <h3 class="title">Free for artists</h3>
-                <h5 class="">Distribution, Funding, Payment Splits, and everything else you'll ever need</h5>
+                <h3 class="title">Start Investing</h3>
+                <h5>Buy shares in music. Create your muisc portfolio. Earn Money</h5>
                 <div class="center extra-padding">
-                    <a class="button sign-up uppercase" href="{% url 'artist_application' %}">Apply Now</a>
+                    <a class="button sign-up uppercase" href="{% url 'register' %}?next={{ request.get_full_path }}">Sign Up</a>
                 </div>
             </div>
             <div>
-                <h3 class="title">$1 per trade</h3>
-                <h5>Buy shares in music projects with just one click</h5>
+                <h3 class="title">For Artists</h3>
+                <h5 class="">Funding, Distribution, Payments, and everything else you'll ever need</h5>
```

Comment:
nit: Remove empty class attribute:
```suggestion
                <h5>Funding, Distribution, Payments, and everything else you'll ever need</h5>
```